vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -38,7 +43,22 @@
      */
     @Override
     public ExhibitionAboutPage store(ExhibitionAboutPage exhibitionAboutPage) {
+        IExhibition exhibition = exhibitionManager.getStartExhibition();
+        if(!exhibition.isAboutPageConfigured()) {
+            exhibition.setAboutPageConfigured(true);
+            exhibitionManager.storeExhibition((Exhibition)exhibition);
+        }
         return repo.save(exhibitionAboutPage);
     }
     
+    /**
+     *  This method returns the first Exhibition About Page set by a Staff, if empty
```

Comment:
"if empty" doesn't mean much. what is empty? you mean if there is no exhibition object yet, don't you? Also, this needs to be proper javadoc (return value)